test/libyul/yulOptimizerTests/fullSuite/storage.yul

```diff
@@ -0,0 +1,15 @@
+{
+    sstore(4, 5)
+    sstore(4, 3)
+    sstore(8, sload(4))
+}
+// ====
+// step: fullSuite
+// ----
+// {
+//     {
+//         sstore(4, 5)
```

Comment:
Why is this one not removed?

Reply:
Redundant store removal is something else, which is probable much more complicated (take a look at the RedundantAssignEliminator). This is just about redundant load elimination.